metadata.rb

```diff
@@ -4,6 +4,9 @@
 license          'GNU Public License 3.0'
 description      'Installs/Configures zabbix_ng'
 long_description 'Installs/Configures zabbix_ng'
-version          '0.1.1'
+version          '0.1.3'
```

Comment:
Remove the version bump altogether, as with this, `0.1.2` will never be in the supermarket. I think it's better to just leave the version bump itself to the maintainer.